Dockerfile

```diff
@@ -1,17 +1,9 @@
-FROM node:10.15.3
+FROM node:14
```

Comment:
Any reason for updating the version of node?

Reply:
no i just ran this version of node during FEC and was comfortable with it when making alterations to the legacy Dockerfile.